src/service/arguments/action/action-arguments.ts

```diff
@@ -0,0 +1,93 @@
+import { FlowType, InputValues, LoggerLevel } from "@bc/domain/inputs";
+import * as core from "@actions/core";
+import { OptionValues } from "commander";
+import Container from "typedi";
+import { ParsedInputs } from "@bc/service/inputs/parsed-inputs";
+import { InvalidInput } from "@bc/domain/errors";
+
+/**
+ * Parses all inputs from github action workflow files
+ */
+export class ActionArguments {
+    /**
+     * Converts user input to corresponding FlowType enum
+     * @param flowType value for "flow-type" input
+     * @returns corresponding enum
+     */
+    private getFlowType(flowType: string): FlowType {
```

Comment:
it is a chance here to refactor this method to
```
     private getFlowType(flowType: string): FlowType {
          return FlowType[flowType] as FlowType;
     }
```
you will have to deal with exception and align `flowType` argument values with FlowType enum values.
Up to you

Reply:
Wouldn't you want it to throw an error since the argument `flowType` is received from the user?

Reply:
stupid Typescript!!! I tried several options and yours seems to be the best!!